spark/src/main/resources/error/delta-error-classes.json

```
@@ -2719,6 +2719,12 @@
     ],
     "sqlState" : "0AKDC"
   },
+  "DELTA_UNSUPPORTED_CATALOG_OWNED_PROPERTY" : {
```

`UNSUPPORTED_CATALOG_OWNED_PROPERTY` sounds a bit weird to me, we would like to block the CO table **_creation_** rather than the property, right? cc @junlee-db.

A clearer alternative maybe:
```suggestion
  "DELTA_UNSUPPORTED_CATALOG_OWNED_TABLE_CREATION" : {
```